src/interfaces/IMetaMorpho.sol

```diff
@@ -30,57 +30,148 @@ interface IOwnable {
 /// @dev This interface is used for factorizing IMetaMorphoStaticTyping and IMetaMorpho.
 /// @dev Consider using the IMetaMorpho interface instead of this one.
 interface IMetaMorphoBase {
```

Comment:
@MerlinEgalite please verity that **each** function is properly and fully natspec documented with at least `@notice` and all the needed `@param` and `@return` tags. Many of the functions are missing these tags.